### PR TITLE
Add wait for controller method

### PR DESCRIPTION
**Pull request by iovadia**
Signed-off-by: iovadia <iovadia@redhat.com>

Used in tests to wait for controller and cause cluster error status

### CODE REVIEW

**Review by YuviGold — APPROVED** [review]

File: discovery-infra/test_infra/logs_utils.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 NUM_OF_RETRIES = 6
 
 
-def verify_logs_uploaded(cluster_tar_path, expected_min_log_num, installation_success, check_oc=False):
+def verify_logs_uploaded(cluster_tar_path, expected_min_log_num, installation_success, verify_control_plane, check_oc=False):
```
Comment (YuviGold) [inline discovery-infra/test_infra/logs_utils.py:14]: Please set `verify_control_plane` with a default value so the API won't get broken.
For example, this function is being called from [download_logs module](https://github.com/openshift/assisted-test-infra/blob/5c615a2b85b2c7b0a5867d75bf944463e0f08745/discovery-infra/download_logs.py#L133-L135) and also on the [d/s tests](https://gitlab.cee.redhat.com/ocp-edge-qe/kni-assisted-installer-auto/blob/9bcbbfc15586cb0e782d003e27b05533c8056cb9/api_tests/test_logs.py#L33)

**Review by YuviGold — CHANGES_REQUESTED** [review]

File: discovery-infra/test_infra/logs_utils.py (hunk 1 above)
Comment (iovadia) [reply discovery-infra/test_infra/logs_utils.py:14]: Thank you for the review, it has a default value.
https://github.com/openshift/assisted-test-infra/pull/434/files#diff-c389358d0828525582921b728933e4e8fac124f747d6bc1a955f4edf2ae3667aR73

Comment (YuviGold) [reply discovery-infra/test_infra/logs_utils.py:14]: Only the inner private function has a default value, while the public function is missing one.

Comment (iovadia) [reply discovery-infra/test_infra/logs_utils.py]: Got it, thanks

**Review by YuviGold — APPROVED** [review]

**Pull request merged**